huaweicloud/services/acceptance/dws/resource_huaweicloud_dws_cluster_test.go

```diff
@@ -232,6 +270,52 @@ resource "huaweicloud_dws_cluster" "test" {
 `, baseNetwork, rName, numberOfNode, password, publicIpBindType, volumeCap, tag)
 }
 
+func testAccDwsCluster_basicV2_mutilAZs(rName string, numberOfNode int, publicIpBindType, password, tag string, volumeCap int) string {
+	baseNetwork := common.TestBaseNetwork(rName)
+
+	return fmt.Sprintf(`
+%s
+
+data "huaweicloud_availability_zones" "test" {}
+
+data "huaweicloud_dws_flavors" "test" {
+  vcpus = 4
+  memory = 16
+  datastore_type = "dws"
+}
+
+resource "huaweicloud_dws_cluster" "test" {
+  name              = "%s"
+  node_type         = "dwsk3.4U16G.4DPU"
+  number_of_node    = %d
+  vpc_id            = huaweicloud_vpc.test.id
+  network_id        = huaweicloud_vpc_subnet.test.id
+  security_group_id = huaweicloud_networking_secgroup.test.id
+  availability_zone =  join(",", [data.huaweicloud_availability_zones.test.names[0],
+                                  data.huaweicloud_availability_zones.test.names[1],
+                                  data.huaweicloud_availability_zones.test.names[3]])
```

Comment:
If there are special combination requirements for az, it is recommended to use environment variable substitution.
Then add usage condition restrictions in the environment variables.

Reply:
done